examples/typescript-server/app.ts

```diff
@@ -0,0 +1,12 @@
+import fastify from '../../fastify'
+import { registerRoutes } from './plugins/routes'
+
+const createServer = () => {
+  const server = fastify()
+
+  registerRoutes(server)
```

Comment:
To me, it is a weird styles to passing the `instance` itself for routes registration.
And it also nested two level of function for a single route.

I recommend to contribute with https://github.com/fastify/demo

Reply:
Hi @climba03003, 

Agreed with you point of view. I focused more in the typescript part, I will recheck and try to follow the pattern on fastify/demo.